src/morphocut/contrib/zooprocess.py

```diff
@@ -92,6 +95,40 @@ def regionprop2zooprocess(prop):
         "solidity": prop.solidity,
     }
 
+def calculateFeret(image):
+    boundaries = skimage.segmentation.find_boundaries(image, mode="inner") #find the boundary
+    border_coordinates = np.nonzero(boundaries) #find the boundary coordinates
+    min_diam, max_diam = calculate_diameter(border_coordinates)
+
+    return min_diam, max_diam
+
+def calculateConvexHulls(border_coordinates):
+    hulls = ConvexHull(border_coordinates)
```

Comment:
The rest of this function calculates the hull, you shouldn't need `scipy.spatial.ConvexHull` at all.